addons/hr_homeworking_calendar/static/src/calendar/common/calendar_common_renderer.js

```diff
@@ -101,13 +125,13 @@ patch(AttendeeCalendarCommonRenderer.prototype, {
         }
     },
     headerTemplateProps(date) {
-        if (this.props.model.scale === "month") {
-            return super.headerTemplateProps(date);
-        }
         const parsedDate = DateTime.fromJSDate(date).toISODate();
         const multiCalendar = this.props.model.multiCalendar;
```

Comment:
you can move this before the workLocationSetForCurrentUser since it is not used before the potential return